src/main/java/com/revature/controllers/EvalController.java

```diff
@@ -69,4 +72,92 @@
 		return ResponseEntity.ok(evalLogic.getPersonEvalsByWeek(pageable,id,num,type));
 	}
 	
+//EVAL CUD---------------------------------------------
+	@RequestMapping(method = RequestMethod.POST, value = "/add")
+	public ResponseEntity<Eval> addEval(@RequestBody Eval eval){
+		
+		return ResponseEntity.ok(evalLogic.createEval(eval));
+	}
+	
+	@RequestMapping(method = RequestMethod.PUT, value = "/update/{id}")
+	public ResponseEntity<Eval> updateEval(@RequestBody Eval eval, @PathVariable ("id") Integer id){
+		
+		Eval currEval = evalLogic.getEvalById(id);
+		
+		if(currEval == null){
+			System.out.println("Eval with id " + id + " not found");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS106)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal " not found" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1192)